upload.linq

```diff
@@ -88,7 +91,7 @@ let uploadFiles (files : seq<FileInfo>) userId token (checksums:seq<string>) =
             dc.Content <- sprintf "%s\n (%d of %d)" file.FullName count total
             Util.Progress <- System.Nullable (int ((count ./. total)*100.0))
             not (contains md checksums)
-    let missing = files |> Seq.filter isMissing
+    let missing = files |> pFilter isMissing
```

Comment:
or maybe we should add `|> List.ofSeq` to just execute immediately and not get the upload function involved until after the filtering is done...